grpclog/logger.go

```diff
@@ -39,19 +39,14 @@ package grpclog // import "google.golang.org/grpc/grpclog"
 import (
 	"log"
 	"os"
-
-	"github.com/golang/glog"
 )
 
-var (
-	// GLogger is a Logger that uses glog. This is the default logger.
-	GLogger Logger = &glogger{}
-
-	// StdLogger is a Logger that uses golang's standard logger.
-	StdLogger Logger = log.New(os.Stderr, "", log.LstdFlags)
+var logger Logger
 
-	logger = GLogger
-)
+func init() {
```

Comment:
I would prefer a variable initialization instead of init().

var logger = log.New(os.Stderr, "", log.LstdFlags)